### PR TITLE
LAB-2089 add ignore bad normals option

### PR DESCRIPTION
**Pull request by katSchmid**

### CODE REVIEW

File: lib/loadObj.js [hunk 1]
```diff
@@ -213,11 +213,11 @@ function loadObj(objPath, options) {
     }
   }
 
-  function createVertex(p, u, n) {
+  function createVertex(p, u, n, ignoreBadNormals = false) {
     // Positions
     if (defined(p) && globalPositions.length > 0) {
       if (p * 3 >= globalPositions.length) {
-        throw new RuntimeError(`Position index ${p} is out of bounds`);
+        //throw new RuntimeError(`Position index ${p} is out of bounds`);
```
Comment (chris-cooper) [inline lib/loadObj.js]: Should this also be wrapped in `if (!ignoreBadNormals) {` rather than commented out?

Comment (katSchmid) [reply lib/loadObj.js]: sorry yes, that was the point of it all

**Review by chris-cooper — APPROVED** [review]
Looks like most other changes are already in main

**Comment by katSchmid** [comment]
I roll main back otherwise, the history isnt clean

**Pull request merged**